schematics/translator.py

```diff
@@ -0,0 +1,11 @@
+class Translator():
+    real_translator = staticmethod(lambda message, *args, **kwargs: message)
+
+    def __call__(self, *args, **kwargs):
+        return self.real_translator(*args, **kwargs)
+
+    def register_translator(self, new_translator):
+        self.real_translator = new_translator
+
+_ = Translator()
```

Comment:
Are you talking about the `_`, that's normally the pattern for translation callable, same thing for Babel for example